src/main/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/SealightsCLIBuildStep.java

```diff
@@ -145,8 +177,8 @@ public boolean perform(AbstractBuild<?, ?> build, Launcher launcher, BuildListen
 
             CLIHandler cliHandler = new CLIHandler(logger);
             cliRunner = createCLIRunner(commandMode);
-
-            isStepSuccessful = cliRunner.perform(build, launcher, listener, commandMode, cliHandler, logger);
+            LogConfiguration logConfiguration = new LogConfiguration(logFolder, logDestination, logLevel);
```

Comment:
I suggest to create it one line before as that's in the middle of a flow.

Reply:
Done